### PR TITLE
Improvement/expose storage event control feature

### PR DESCRIPTION
**Pull request by azriel91**
## Additions

- `"storage-event-control"` feature enables the `specs` `"storage-event-control"` feature.

## PR Checklist

By placing an x in the boxes I certify that I have:

- **n/a** Updated the content of the book if this PR would make the book outdated.
- [x] Added a changelog entry if this will impact users, or modified more than 5 lines of Rust that wasn't a doc comment.
- **n/a** Added unit tests for new code added in this PR.
- [x] Acknowledged that by making this pull request I release this code under an MIT/Apache 2.0 dual licensing scheme.

If this modified or created any rs files:

- **n/a** Ran `cargo +stable fmt --all`
- **n/a** Ran `cargo clippy --all --features "empty"`
- **n/a** Ran `cargo test --all --features "empty"`

### CODE REVIEW

**Comment by azriel91** [comment]
Weird servo compilation bug on nightly (and why are we depending on servo?):

```
error: failed to run custom build command for `servo-fontconfig-sys v4.0.9`

Caused by:
  process didn't exit successfully: `/home/jenkins/workspace/se-storage-event-control-feature@2/target/debug/build/servo-fontconfig-sys-e91e747806accdc6/build-script-build` (exit code: 101)
--- stdout

# .. elided

conftest.c:85:12: fatal error: atomic.h: No such file or directory
   #include <atomic.h>
            ^~~~~~~~~~
compilation terminated.
```

bors try

File: Jenkinsfile [hunk 1]
```diff
@@ -40,9 +40,9 @@ pipeline {
                     steps {
                         sh 'cargo update'
                         // Perform actual check
-                        sh 'cargo check --all --all-targets --features "vulkan sdl_controller json saveload tiles"'
+                        sh 'cargo check --all --all-targets --features "vulkan sdl_controller json saveload tiles storage-event-control"'
```
Comment (TimonPost) [inline Jenkinsfile]: Are those all the features amethyst has? We can also just do --all-features to enable all feature flags.

Comment (azriel91) [reply Jenkinsfile]: we can't unfortunately -- because of `"vulkan"` / `"empty"` / `"metal"` :art:

**Comment by azriel91** [comment]
bors retry

**Pull request merged**